yb-voyager/cmd/importData.go

```diff
@@ -1660,6 +1666,10 @@ func cleanMSRForImportDataStartClean() error {
 	return nil
 }
 
+func isPrimaryKeyConflictIgnoreModeValid() bool {
```

Comment:
`isPrimaryKeyConflictValid` ? 
It's not specific to IGNORE, right? Even in case of ERROR, we'd like to to store and validate, right?